pages/api/hello.ts

```diff
@@ -1,13 +1,28 @@
 // Next.js API route support: https://nextjs.org/docs/api-routes/introduction
```

Comment:
hmmm I have a slightly hard time understanding how hello is related to this file with what looks like google auth fetching. It may be good to clarify